src/full/Agda/Interaction/Options/Base.hs

```diff
@@ -348,7 +348,7 @@ data PragmaOptions = PragmaOptions
       -- ^ Allow definitions by copattern matching?
   , _optPatternMatching           :: WithDefault 'True
       -- ^ Is pattern matching allowed in the current file?
-  , _optExactSplit                :: WithDefault 'False
+  , _optExactSplit                :: WithDefault 'True
```

Comment:
This seems insufficient to change the default.
One has to invoke `exactSplitFlag True`: https://github.com/agda/agda/blob/c436dd4cf5e42390cecec88cc7f7f6978ff8d1c4/src/full/Agda/Interaction/Options/Base.hs#L1469-L1472